test/e2e/magellan-ie11-canary.json

```diff
@@ -1,5 +1,5 @@
 {
-	"mocha_tests": [ "specs/" ],
+	"mocha_tests": [ "specs/specs-calypso" ],
```

Comment:
I feel like we can remove this ie11 file entirely since we no longer run any IE11 cases?

Reply:
Yeah, most likely.

I was planning to do a follow up PR removing unused `magellan-*` configs plus the flag in `run.sh` that uses them.